repo/scaffold/application.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from ramble.appkit import *
+
+
+class Scaffold(ExecutableApplication):
+    """Scaffold benchmark"""
+
+    name = "scaffold"
+
+    tags = ["python"]
+
+    executable(
+        "modules",
+        "ml load rocm/6.2.1 rocmcc/6.2.1-cce-18.0.1a-magic",
+    )
+    executable(
+        "build",
+        "pip install {package_path} --extra-index-url https://download.pytorch.org/whl/rocm6.2",
+        output_capture=OUTPUT_CAPTURE.ALL,
+    )
```

Comment:
How can I tie this to `ramble workspace setup` pipeline?

Reply:
I think I figured this out with [5c02432](https://github.com/LLNL/benchpark/pull/687/commits/5c02432fcb120a32e1698ed155719a9435fd9c44), specifying the spec a different way.